requirements/requirements.in

```diff
@@ -7,6 +7,7 @@ celery==4.2.1
 daphne==1.3.0   # Last before backwards-incompatible channels 2 upgrade
 Django==1.11.16
 django-auth-ldap==1.2.8
+django-cors-headers==2.4.0
```

Comment:
This isn't quite enough to make this work.  Please give this doc a read:

https://github.com/ansible/awx/blob/devel/requirements/README.md

Reply:
Thanks for the info. I'll check it out and make changes accordingly.

Reply:
I updated `requirements/requirements.txt` with the appropriate dependencies. Turns out that it's just `django-cors-headers==2.4.0` as all of its deps are for testing only.